src/install/demo/intro.mdx

```diff
@@ -0,0 +1,8 @@
+---
+headingText: Branching doc demo
+componentType: default
+---
+
+This is a demo branching doc. You can find the content for this doc at `src/install/demo` in our repo.
+
+Click the options to see how content is swapped in and out.
```

Comment:
@paperclypse - 

I think you could write something like:

"Below you will find a demo branching doc. You can find the content for this doc at `src/install/demo` in our repo.

As you explore the demo branching doc below, click the various options to see how content is swapped in and out."

Reply:
Works for me!